### DISCUSSION TITLE
PANEL: Comunidades tecnológicas alrededor de América Latina

### DISCUSSION THREAD

**Discussion by David-Lazaro-Fernandez** [gittogethers/octogatosconf22#83] [Talks in Spanish]
## Host: 

> Denny Perez

## Panelists: 

> Vanessa Marely Aristizabal
> Andres Pineda
> Dorian Hidalgo
> Ramón Huidobro

**Comment by elchamojavier** [comment]
P: En America Latina, tienen programas de Mentores/as en sus comunidades??  Y cual es la mejor forma de contribuir??

**Reply by dahidalgo** [reply]
Formas de contribuir hay varias, puedes dar charlas, ayudar en la organización de algún evento, moderando las redes de la comunidad, traduciendo documentación. Según vayas involucrándote en alguna comunidad vas viendo que necesidades se van dando, que podría mejorar, etc.

**Reply by ajpinedam** [reply]
Conozco una comunidad, o mejor dicho, un hub de comunidades Python en Latam que tienen este programa de mentoria. Estos son https://hablemospython.dev/. 

Tienen un server de discord donde se pueden unir y desde ahi conectar con los diferentes programas que ofrecen.

Y sobre contribuir ese servidor es para que todos aprendamos juntos, pueden empezar a sus contribuciones ahi mismo o en iniciativas que salen de ese grupo. 

100% recomendado.
